Resources/Prototypes/Entities/Mobs/Player/humanoid.yml

```
@@ -45,6 +45,8 @@
       description: ghost-role-information-Death-Squad-description
       raffle:
         settings: short
+      mindRoles:
+      - MindRoleGhostRoleFamiliar
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)